server/data/prisonRegisterClient.ts

```diff
@@ -13,9 +13,10 @@ export default class PrisonRegisterClient {
     })
   }
 
-  async getAllPrisons(token: string): Promise<PrisonResponse> {
-    return PrisonRegisterClient.restClient(token).get({
+  async getAllPrisons(token: string): Promise<Array<PrisonResponse>> {
```

Comment:
Whoops! I used the wrong method signature when I first wrote this! It only came out of the woodwork now we are using it! 😁